### DISCUSSION TITLE
Reset the VM password without needing to shut it down

### DISCUSSION THREAD

**Discussion by tuanhoangth1603** [apache/cloudstack#9388] [Q&A] [answered]
Hello everyone,

Can CloudStack v19 reset the VM password without needing to shut it down? Or is there any other way to accomplish this?

**Comment by yadvr** [comment, marked as answer]
By design it is not possible currently @tuanhoangth1603 because we don't run any agent in the guest VMs that can get real-time changes; the VM only gets the password/metadata (via cloud-init etc.) from the config drive or VR when the VM boots.

**Comment by tuanhoangth1603** [comment]
Thanks for your answer. Do we have a roadmap for this one?
